docs/guides/wireguard/client.md

```diff
@@ -6,7 +6,7 @@ For each new client, the following steps must be taken. For the sake of simplici
 
 <!-- markdownlint-disable code-block-style -->
 ??? info "All commands described below at once"
-    ``` bash
+    ```bash
```

Comment:
Does this matter really? Spec is that the infostring is the first word after the triple ticks. Extra space "shouldn't" matter?

Reply:
It doesn't matter, but it's inconsistent with the codebase; we don't use spaces anywhere else apart from this branch.